### DISCUSSION TITLE
Pimcore (Packaged by Bitnami) Beginner Help: Uploading Suitable Template

### DISCUSSION THREAD

**Discussion by skylerchaikin** [pimcore/pimcore#15901] [Q&A / Support]
Hello! 
I have been tasked with setting up a pimcore server for a startup (a recording studio) that I work at. We are trying to use Pimcore to develop a very simple Digital Asset Manager, where we can upload our assets, like video and audio recordings, in order to organize them and share them with our clients. 

I have started out with the Pimcore Demo packaged by Bitnami, running on a VirtualBox VM. 

**Problem 1: I do not know of a way to easily and quickly delete the Demo files so that I can turn the Demo version into the Skeleton version.** If there is some terminal command that I could use, or some bundle or file somewhere that I could delete to solve this, that would make things a lot easier. I have just been deleting all of the assets and documents whenever I need to set up a new pimcore sever (I have had to restart several times for various reasons, usually because of weird errors i get that never go away). 

**Problem 2: I wish to upload some sort of template or module that could turn my pimcore server into something that already resembles the type of simple DAM that I am looking for, if such a template or method exists.** I have done a bit a research about the pimcore templates, but all of the solutions I am finding involve programming (forgive me, my programming knowledge is limited). So, essentially, my supervisor is looking for a more friendly way to upload some sort of template that would be to our benefit. 

**Problem 3: Often, pretty much every other day, I have to start from square 1 because I get an error that says that the "server threw an exception", when I try to "Save and Publish", and as far as I can tell there is no way of bypassing this error.** 

 If anyone could help guide me in these two issues, or give me any guidance in any way, that would be great.

**Comment by kingjia90** [comment]
1: It looks like these package are done by Bitnami itself, https://docs.bitnami.com/general/apps/pimcore/#:~:text=This%20software%20listing%20is%20packaged%20by%20Bitnami. 
I think i have never heard about maintaining this internally in my 1 and half + years working here.

2: mhhh, i am glad to hear that is resembling what you are looking for, but please have a look to https://pimcore.com/docs/platform/Pimcore/Overview/#:~:text=Pimcore%20is%20built%20for%20developers,a%20developer%20to%20get%20started.  without a PHP MVC dev would be quite hard, even by having one, it should be a [certified dev](https://pimcore.com/en/developers/academy).

3: on this, i can't help much, that's a generic error, it could be everything, we would need the logs, stack trace or any lines mentioned in there and specific way to reproduce it. Probably something from problem 1 was not properly cleaned, but at the end of the day, it is something that a dev would know right away how to deal with it.
